AUTHORS

```diff
@@ -543,3 +544,64 @@ Dennis Meckel <meckel@datenschuppen.de>
 Harshil Meena <harshil.7535@gmail.com>
 Micky <mickydroch@gmail.com>
 arghdos <arghdos@gmail.com>
+Jainul Vaghasia <jainulvaghasia@D-10-157-53-226.dhcp4.washington.edu>
+Michele Zaffalon <michele.zaffalon@gmail.com>
+Martha Giannoudovardi <maapxa@gmail.com>
+devangkulshreshtha <devang.kulshreshtha.cse14@itbhu.ac.in>
+steph papanik <spapanik21@gmail.com>
+Isuru Fernando <isuru.11@cse.mrt.ac.lk>
+msdousti <msdousti@gmail.com>
+Arif Ahmed <arif.ahmed.5.10.1995@gmail.com>
+Abdullah Javed Nesar <abduljaved1994@gmail.com>
+Lakshya Agrawal <zeeshan.lakshya@gmail.com>
+shruti <shrutishrm512@gmail.com>
+ArifAhmed1995 <arif.ahmed.5.10.1995@gmail.com>
+rohitrango <rohit.rango@gmail.com>
```

Comment:
You should fix these using `.mailmap` if possible. https://github.com/rohitrango